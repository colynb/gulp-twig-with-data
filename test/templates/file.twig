<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>test {{ title|default('no options') }}</title>
</head>
<body>

</body>
</html>
