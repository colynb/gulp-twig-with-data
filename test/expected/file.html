<!DOCTYPE html>
<html lang="en">
<head>
    <meta charset="UTF-8">
    <title>test twig</title>
</head>
<body>

</body>
</html>
